apps/biobin/resources/usr/bin/biobin-permute-collate.py

```diff
@@ -5,74 +5,66 @@
 
 
 if (len(sys.argv) < 3) or ("%d" not in sys.argv[2]):
```

Comment:
I still like using argparse for option parsing, even for simple scripts like this. But it's not a big issue.

Also, I suggest to use python3 compatible syntax wherever possible to make conversion easier in the figure. So print statement should be converted into function calls (i.e. print(...)).